Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -141,31 +141,49 @@ final class BuildPlanTests: XCTestCase {
+        XCTAssertMatch(lib, ["-swift-version", "4", "-enable-batch-mode", "-Onone", "-enable-testing", "-g", .equal(j), "-DSWIFT_PACKAGE", "-DDEBUG", "-module-cache-path", "\(objroot.appending(components: "ModuleCache"))", .anySequence])
 
       #if os(macOS)
         let linkArguments = [
-            "/fake/path/to/swiftc", "-L", "/path/to/build/debug",
-            "-o", "/path/to/build/debug/exe", "-module-name", "exe",
+            result.plan.buildParameters.toolchain.swiftCompilerPath.pathString,
+            "-L", objroot.pathString,
+            "-o", objroot.appending(components: "exe").pathString,
+            "-module-name", "exe",
             "-emit-executable",
             "-Xlinker", "-rpath", "-Xlinker", "@loader_path",
-            "@/path/to/build/debug/exe.product/Objects.LinkFileList",
+            "@\(objroot.appending(components: "exe.product", "Objects.LinkFileList"))",
             "-Xlinker", "-rpath", "-Xlinker", "/fake/path/lib/swift-5.5/macosx",
-            "-target", defaultTargetTriple, "-Xlinker", "-add_ast_path",
-            "-Xlinker", "/path/to/build/debug/exe.build/exe.swiftmodule", "-Xlinker", "-add_ast_path",
-            "-Xlinker", "/path/to/build/debug/lib.swiftmodule",
+            "-target", defaultTargetTriple,
+            "-Xlinker", "-add_ast_path", "-Xlinker", objroot.appending(components: "exe.build", "exe.swiftmodule").pathString,
+            "-Xlinker", "-add_ast_path", "-Xlinker", objroot.appending(components: "lib.swiftmodule").pathString,
+        ]
+      #elseif os(Windows)
+        let linkArguments = [
+            result.plan.buildParameters.toolchain.swiftCompilerPath.pathString,
+            "-L", objroot.pathString,
+            "-o", objroot.appending(components: "exe.exe").pathString,
+            "-module-name", "exe",
+            // "-static-stdlib",
+            "-emit-executable",
+            "@\(objroot.appending(components: "exe.product", "Objects.LinkFileList"))",
+            "-target", defaultTargetTriple,
         ]
       #else
         let linkArguments = [
-            "/fake/path/to/swiftc", "-L", "/path/to/build/debug",
-            "-o", "/path/to/build/debug/exe", "-module-name", "exe",
-            "-static-stdlib", "-emit-executable",
+            result.plan.buildParameters.toolchain.swiftCompilerPath.pathString,
+            "-L", objroot.pathString,
+            "-o", objroot.appending(components: "exe").pathString,
+            "-module-name", "exe",
+            "-static-stdlib",
+            "-emit-executable",
             "-Xlinker", "-rpath=$ORIGIN",
-            "@/path/to/build/debug/exe.product/Objects.LinkFileList",
+            "@\(objroot.appending(components: "exe.product", "Objects.LinkFileList"))",
             "-target", defaultTargetTriple,
         ]
```

Comment:
^^ this is nice cleanup, thanks! one nit: can we call `objroot` `outputPath` or `buildPath`?

Reply:
hahaha ... I think that we cross'ed paths in the middle.  I went from `build` (aka `buildPath`) to `objroot`.  I don't care enough tbh, if you feel that `buildPath` is better, I can do `s/objroot/buildPath/`.  I think that objroot is pretty common for the XCode builds and for Makefile, so its more familiar, but I'm not really invested in the name per se.

Reply:
since we both intuitively went for `buildPath`, I say we go with that